### PR TITLE
Trade Post Cultural Buildings

### PR DESCRIPTION
**Pull request by mdbpr-iscteiulpt**
Heres the completed pull request for the cultural buildings for the trade posts in warcraft:

## Changelog:
- Cultural:
  - Road Tavern (General)
  - Zeppelin Tower (Goblins)
  - Resting Lounge (Night Elves)
  - Jewelcrafters (Humans, blood elves and Draenei)
  - Kodo Breeders (Tauren and Taunka)
  - Turtle Pools (Tuskar)
  - Guide Eagle Nests (Harpies)
  - Wolf Bestiary (Orcs)
  - Dwarven Stonemasons (Dwarves)
  - Chandlers Guild (Kobolts)
  - Merchant Bots (Gnomes)
  - Murloc Coralmasons (Murlocs)
- Religious:
  - Portal Center (Arcane)

## Shorter Version:
- Added new cultural specific buildings for various cultures, and 1 religion specific (Arcane religion).

### CODE REVIEW

File: localisation/00_ciwc_united_localisation.csv [hunk 1]
```diff
@@ -17,4 +17,19 @@ great_sea_trade_route_tooltip;Can only build 
...
+tp_jewelcrafters;Jewelcrafters;;;;;;;;;;;;;x
+tp_kodo_breeders;Kodo Breeders;;;;;;;;;;;;;x
+tp_turtle_pools;Turtle Pools;;;;;;;;;;;;;x
+tp_guide_eagle_nests;Guide Eagle Nests;;;;;;;;;;;;;x
+tp_wolf_bestiary;Wolf Bestiary;;;;;;;;;;;;;x
+tp_voodo_charm_vendor;Voodoo Charm Vendor;;;;;;;;;;;;;x
+tp_dwarfish_stonemasons;Dwarfish Stonemasons;;;;;;;;;;;;;x
+tp_chandlers_guild;Chandlers Guild;;;;;;;;;;;;;x
+tp_merchant_bots;Merchant Bots;;;;;;;;;;;;;x
+tp_murloc_coralmasons;Murloc CoralMasons;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tp_murloc_coralmasons;Murloc Coralmasons;;;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 2]
```diff
@@ -17,4 +17,19 @@ great_sea_trade_route_tooltip;Can only build 
...
+tp_road_tavern;Road Tavern ;;;;;;;;;;;;;x
+tp_zeppelin_tower;Zeppelin Tower ;;;;;;;;;;;;;x
+tp_resting_lounge;Resting Lounge ;;;;;;;;;;;;;x
+tp_jewelcrafters;Jewelcrafters;;;;;;;;;;;;;x
+tp_kodo_breeders;Kodo Breeders;;;;;;;;;;;;;x
+tp_turtle_pools;Turtle Pools;;;;;;;;;;;;;x
+tp_guide_eagle_nests;Guide Eagle Nests;;;;;;;;;;;;;x
+tp_wolf_bestiary;Wolf Bestiary;;;;;;;;;;;;;x
+tp_voodo_charm_vendor;Voodoo Charm Vendor;;;;;;;;;;;;;x
+tp_dwarfish_stonemasons;Dwarfish Stonemasons;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tp_dwarfish_stonemasons;Dwarven Stonemasons;;;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 3]
```diff
@@ -17,4 +17,19 @@ great_sea_trade_route_tooltip;Can only build 
...
+tp_chandlers_guild;Chandlers Guild;;;;;;;;;;;;;x
+tp_merchant_bots;Merchant Bots;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]: Are these actual robots? If not, `boats`.

Comment (mdbpr-iscteiulpt) [reply localisation/00_ciwc_united_localisation.csv]: Its gnomish cultural so yes they are bots

File: localisation/00_ciwc_united_localisation.csv [hunk 4]
```diff
@@ -17,4 +17,21 @@ great_sea_trade_route_tooltip;Can only build 
 ;;;;;;;;;;;;;;x
 text_effect_major_trade_route;This is �GMajor�! Trade Route;;;;;;;;;;;;;x
 text_effect_medium_trade_route;This is �YMedium�! Trade Route;;;;;;;;;;;;;x
-text_effect_minor_trade_route;This is �RMinor�! Trade Route;;;;;;;;;;;;;x
\ No newline at end of file
+text_effect_minor_trade_route;This is �RMinor�! Trade Route;;;;;;;;;;;;;x
```
Comment (arithon) [inline localisation/00_ciwc_united_localisation.csv]: Shouldn't it be "This is **a** Minor/Medium/Major Trade Route?"

Comment (mdbpr-iscteiulpt) [reply localisation/00_ciwc_united_localisation.csv]: Oh this isjt from my code this is from base game i believe

Comment (arithon) [reply localisation/00_ciwc_united_localisation.csv]: could be, but who are we **not** to improve vanilla game where we find problems/mistakes?^^

Comment (MemeWizard) [reply localisation/00_ciwc_united_localisation.csv]: First, it's my code from previous trade branch.
Second, it's not part of this branch, if you want to change it - do it on branch for localization improvements,

Comment (DukeJeans) [reply localisation/00_ciwc_united_localisation.csv]: @arithon #951

Comment (mdbpr1-iscteiulpt) [reply localisation/00_ciwc_united_localisation.csv]: I think ive said before but this is not part of my mod but something that was there before ui believe from a previous pr but i can fix that in this one

Comment (DukeJeans) [reply localisation/00_ciwc_united_localisation.csv]: No need. Accounted for in ca70621.

File: common/buildings/wc_cultural_tradepost.txt [hunk 5]
```diff
@@ -0,0 +1,314 @@
+trade_post = {
+	
+	############CULTURAL#################
+	
+	### General Building
+	tp_road_tavern = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { not_tradepost_culture_groups = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: `not_tradepost_culture_groups`: such scripted trigger doesn't exist.

File: common/buildings/wc_cultural_tradepost.txt [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	### Goblins!!
+	tp_zeppelin_tower = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			culture_group = goblin_group
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: Maybe this?
```suggestion
			FROM = { culture_group = goblin_group }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 7]
```diff
@@ -0,0 +1,314 @@
...
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}	
+	
+	##Altar of Wealth
+	tp_altar_of_wealth = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { religion = arcane_religion }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: `arcane_religion` has 2 buildings?

File: common/buildings/wc_cultural_tradepost.txt [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
+		build_time = 620
+		ai_creation_factor = 95
+		
+		tech_growth_modifier = 0.125
+		military_techpoints = 0.01
+		tradevalue = 15
+		trade_route_wealth = 1
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: Goblins already have 1 building chain with 3 buildings inside: `radhanite_quarter_1`, `radhanite_quarter_2`, `radhanite_quarter_3`. +1 buildings makes them kinda too OP.

File: common/buildings/wc_cultural_tradepost.txt [hunk 9]
```diff
@@ -0,0 +1,314 @@
+trade_post = {
+	
+	############CULTURAL#################
+	
+	### General Building
+	tp_road_tavern = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 10]
```diff
@@ -0,0 +1,314 @@
...
+		garrison_size = 0.5
+		tradevalue = 20
+		trade_route_wealth = 1
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	### Goblins!!
+	tp_zeppelin_tower = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 11]
```diff
@@ -0,0 +1,314 @@
...
+		military_techpoints = 0.01
+		tradevalue = 15
+		trade_route_wealth = 1
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Nigth Elf Group
+	tp_resting_lounge = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 12]
```diff
@@ -0,0 +1,314 @@
...
+		cultural_techpoints = 0.02
+		tradevalue = 10
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+		
+	##Humans, High elves, Blood elves and Draenei
+	tp_jewelcrafters = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 13]
```diff
@@ -0,0 +1,314 @@
...
+		economic_techpoints = 0.01
+		tradevalue = 30
+		trade_route_wealth = 4		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+
+	##Tauren/Taunka
+	tp_kodo_breeders = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 14]
```diff
@@ -0,0 +1,314 @@
...
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Tuskar Group
+	tp_turtle_pools = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 15]
```diff
@@ -0,0 +1,314 @@
...
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Orc Group
+	tp_wolf_bestiary = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 16]
```diff
@@ -0,0 +1,314 @@
...
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Harpy Group
+	tp_guide_eagle_nests = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 17]
```diff
@@ -0,0 +1,314 @@
...
+		retinuesize = 20
+		tech_growth_modifier = 0.2		
+		tradevalue = 15
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+		
+	##Troll Group
+	tp_voodo_charm_vendor = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 18]
```diff
@@ -0,0 +1,314 @@
...
+		cultural_techpoints = 0.02	
+		tradevalue = 20
+		trade_route_wealth = 2	
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Dwarfish Group
+	tp_dwarfish_stonemasons = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 19]
```diff
@@ -0,0 +1,314 @@
...
+		retinuesize = 10
+		military_techpoints = 0.01	
+		tradevalue = 20
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Kobolt Group
+	tp_chandlers_guild = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 20]
```diff
@@ -0,0 +1,314 @@
...
+		cultural_techpoints = 0.01
+		tradevalue = 20
+		trade_route_wealth = 2			
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Goblins Group
+	tp_merchant_bots = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 21]
```diff
@@ -0,0 +1,314 @@
...
+		monthly_character_piety = 0.01
+		economic_techpoints = 0.01
+		cultural_techpoints = 0.01
+		tradevalue = 20
+		trade_route_wealth = 2			
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Goblins Group
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: It says Goblins though it's gnome building.

File: common/buildings/wc_cultural_tradepost.txt [hunk 22]
```diff
@@ -0,0 +1,314 @@
...
+		economic_techpoints = 0.01
+		cultural_techpoints = 0.01
+		tradevalue = 10		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Murloc Group
+	tp_murloc_coralmasons = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 23]
```diff
@@ -0,0 +1,314 @@
...
+		trade_route_wealth = 1
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	#######Religious################
+	
+	##Arcane religion
+	tp_portal_center = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 24]
```diff
@@ -0,0 +1,314 @@
...
+		military_techpoints = 0.02
+		economic_techpoints = 0.02
+		cultural_techpoints = 0.02
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}	
+	
+	##Altar of Wealth
+	tp_altar_of_wealth = {
+		potential = { in_major_trade_route_trigger = yes }
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: We no longer limit trade post buildings. It's not needed.
```suggestion
		potential = { in_trade_route_trigger = yes }
```

**Comment by mdbpr-iscteiulpt** [comment]
Self review:
- Must move ALL triggers into potential when related to owner culture…. Considering that base game doesnt show buildings that you cant built and not the requiremets you dont have for them

File: common/buildings/wc_cultural_tradepost.txt (hunk 21 above)
Comment (mdbpr1-iscteiulpt) [reply common/buildings/wc_cultural_tradepost.txt]: solved

File: common/buildings/wc_cultural_tradepost.txt (hunk 8 above)
Comment (mdbpr1-iscteiulpt) [reply common/buildings/wc_cultural_tradepost.txt]: Deleted zeeplins completly

File: common/buildings/wc_cultural_tradepost.txt (hunk 7 above)
Comment (mdbpr1-iscteiulpt) [reply common/buildings/wc_cultural_tradepost.txt]: arcane building is basicaly related to travel and tech and provides no direct economy boost. This building was made with idea in mind first then name and theme

File: common/buildings/wc_cultural_tradepost.txt (hunk 5 above)
Comment (mdbpr1-iscteiulpt) [reply common/buildings/wc_cultural_tradepost.txt]: I forgot to create it XD

**Comment by mdbpr1-iscteiulpt** [comment]
> 
> 
> Fixed you say? But you deleted half like for altar if wealth and left unnecessary space in portal building.

Altar of wealth was deleted on porpuse since cult of wealth already  had a building XD

**Comment by mdbpr1-iscteiulpt** [comment]
>  for altar if wealth and left unnecessary space in portal building.

MB the space was for the tr_empty so that all buildings use a empty description since their name is pretty self explanatory on the other hand i fixed the part where i the german loc was still on it but thanks for bringing it up!

File: common/buildings/wc_cultural_tradepost.txt [hunk 25]
```diff
@@ -0,0 +1,314 @@
...
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 275
+		build_time = 180
+		ai_creation_factor = 95
+		
+		garrison_size = 0.5
+		tradevalue = 20
+		trade_route_wealth = 1
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: [The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		trade_route_wealth = 0.2
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 26]
```diff
@@ -0,0 +1,314 @@
...
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 225
+		build_time = 180
+		ai_creation_factor = 95
+		
+		monthly_character_piety = 0.03
+		garrison_size = 0.5
+		tech_growth_modifier = 0.125
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: [The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		tech_growth_modifier = 0.1
```
Plus, `retinuesize = 1` bonus is missing. Though it's too small. For example, `trade_offices_1` gives 50.

File: common/buildings/wc_cultural_tradepost.txt [hunk 27]
```diff
@@ -0,0 +1,314 @@
...
+			FROM = { culture_group = harpy_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 300
+		build_time = 270
+		ai_creation_factor = 95
+		
+		retinuesize = 20
+		tech_growth_modifier = 0.2
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: [The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		tech_growth_modifier = 0.125
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 28]
```diff
@@ -0,0 +1,314 @@
...
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 200
+		build_time = 180
+		ai_creation_factor = 95
+
+		monthly_character_piety = 0.01
+		economic_techpoints = 0.01
+		cultural_techpoints = 0.01
+		tradevalue = 20
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: [The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		tradevalue = 10
```

File: common/buildings/wc_cultural_tradepost.txt [hunk 29]
```diff
@@ -0,0 +1,314 @@
...
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 400
+		build_time = 270
+		ai_creation_factor = 95
+
+		tech_growth_modifier = 0.2
+		cultural_techpoints = 0.01
+		tradevalue = 30
```
Comment (zumbak04) [inline common/buildings/wc_cultural_tradepost.txt]: [The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		tradevalue = 25
```

**Comment by zumbak04** [comment]
Evil religions like Death God, Sargerite, Shath'yar can't build and get bonuses from vanilla trade post buildings, but can from these because blocks (like on the screenshot below) aren't here.
<details>
<summary>Vanilla Buildings</summary>

image

</details>

**Comment by zumbak04** [comment]
@mdbpr1-iscteiulpt , fixed 2 issues in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/950/commits/eb6391dacd2a7a1127f728d338ae4c48ebb46c05 and https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/950/commits/0599386e6673f5be2e3cd0c6e9cc2156c0ef5ca7 . Couldn't find anything beside these ones.

Though, update the Changelog: mention changes of `radhanite_quarter_<1-3>` triggers.

File: localisation/00_ciwc_united_localisation.csv [hunk 30]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
 text_effect_major_trade_route;This is �GMajor�! Trade Route;;;;;;;;;;;;;x
 text_effect_medium_trade_route;This is �YMedium�! Trade Route;;;;;;;;;;;;;x
-text_effect_minor_trade_route;This is �RMinor�! Trade Route;;;;;;;;;;;;;x
\ No newline at end of file
+text_effect_minor_trade_route;This is �RMinor�! Trade Route;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+tr_culture_none;Road Tavern;;Stra�entaverne;;;;;;;;;;;x
+tr_culture_group_goblin;Zeppelin Tower;;Zeppelinturm;;;;;;;;;;;x
+tr_culture_group_night_elf;Resting Lounge;;Erholungslounge;;;;;;;;;;;x
+tr_culture_group_high_elf_human_draenei;Jewellery Store;;Juweliere;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tr_culture_group_high_elf_human_draenei;Jeweler's Shop;;Juweliere;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 31]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
+tr_culture_group_tauren;Kodo Breedery;;Kodo-Z�chter;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tr_culture_group_tauren;Kodo Ranch;;Kodo-Z�chter;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 32]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
+tr_culture_group_tuskarr;Turtle Pools;;Schildkr�tenbecken;;;;;;;;;;;x
+tr_culture_group_harpy;Guide Eagle Nests;;Leitadlernester;;;;;;;;;;;x
+tr_culture_group_orc;Wolf Bestiary;;Wolfsbestiarium;;;;;;;;;;;x
+tr_culture_group_troll;Charm Shop;;Amulettverk�ufer;;;;;;;;;;;x
+tr_culture_group_dwarf;Stonemasonry;;Steinmetzgilde;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tr_culture_group_dwarf;Stonemasons Guild;;Steinmetzgilde;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 33]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
+tr_culture_group_kobold;Chandlers Guild;;Kerzenziehergilde;;;;;;;;;;;x
+tr_culture_group_gnome;Merchant Bots Hub;;Handelsbotzentrum;;;;;;;;;;;x
+tr_culture_group_murloc;Coralmasonry;;Korallenmei�ler;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tr_culture_group_murloc;Coralmasons Guild;;Korallenmei�ler;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 34]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
+;;;;;;;;;;;;;;x
+tr_religion_arcane_religion;Portal Center;;Portalzentrum;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+tr_culture_none_desc;A place where merchants can get a roof over their heads and a plate of food.;;Ein Ort, an dem H�ndler ein Dach �ber dem Kopf und einen Teller mit Essen bekommen k�nnen.;;;;;;;;;;;x
+tr_culture_group_goblin_desc;A building where a zeppelin docks to pickup and drop off passengers.;;Ein Geb�ude, an dem ein Zeppelin anlegt, um Passagiere abzuholen und abzusetzen.;;;;;;;;;;;x
+tr_culture_group_night_elf_desc;A place where travellers of all sorts can rest.;;Ein Ort, an dem sich Reisende aller Art ausruhen k�nnen.;;;;;;;;;;;x
+tr_culture_group_high_elf_human_draenei_desc;In this store jewelers sells their precious masterpieces.;;In diesem Gesch�ft verkaufen Juweliere ihre kostbaren Meisterwerke.;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tr_culture_group_high_elf_human_draenei_desc;In this store, jewelers sell their precious masterpieces.;;In diesem Gesch�ft verkaufen Juweliere ihre kostbaren Meisterwerke.;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 35]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
+tr_culture_group_tauren_desc;Many kodo beasts breeds happily on this farm.;;Auf dieser Farm werden viele gl�ckliche Kodo-Bestien gez�chet.;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]: This is a weird localisation
```suggestion
tr_culture_group_tauren_desc;Many kodo beasts breed happily on this farm.;;Auf dieser Farm werden viele gl�ckliche Kodo-Bestien gez�chet.;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 36]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
+tr_culture_group_tuskarr_desc;Turtles can make it to these water pools.;;Becken voll martialischer Schildkr�ten.;;;;;;;;;;;x
+tr_culture_group_harpy_desc;Sharp-eyed eagles that live here help merchants avoid dangerous paths.;;Scharf�ugige Adler, die hier leben, helfen H�ndlern, gef�hrliche Pfade zu vermeiden.;;;;;;;;;;;x
+tr_culture_group_orc_desc;Big, but not really bad wolves are used here for transporting goods.;;Gro�e, aber nicht wirklich b�se W�lfe werden hier zum Transport von Waren eingesetzt.;;;;;;;;;;;x
+tr_culture_group_troll_desc;Charms against curses, misfortune and greedy merchants is produced and sold here. No returns or refunds. Ever.;;Amulette gegen Fl�che, Ungl�ck und gierige Kaufleute werden hier produziert und verkauft. Keine R�ckgabe oder R�ckerstattung. Niemals.;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tr_culture_group_troll_desc;Charms against curses, misfortune and greedy merchants are produced and sold here. No returns or refunds. Ever.;;Amulette gegen Fl�che, Ungl�ck und gierige Kaufleute werden hier produziert und verkauft. Keine R�ckgabe oder R�ckerstattung. Niemals.;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 37]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
+tr_culture_group_dwarf_desc;Here hard stones are used in even harder mason's work.;;Hier werden harte Steine f�r noch h�rtere Maurerarbeiten verwendet.;;;;;;;;;;;x
+tr_culture_group_kobold_desc;Candlemakers works here as long as nobody don't touch them.;;Kerzenzieher arbeiten hier, solange niemand sie anr�hrt.;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tr_culture_group_kobold_desc;Chandlers work here to produce candles so long as you no take them!;;Kerzenzieher arbeiten hier, solange niemand sie anr�hrt.;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv [hunk 38]
```diff
@@ -17,4 +17,36 @@ great_sea_trade_route_tooltip;Can only build 
...
+tr_culture_group_gnome_desc;Merchant bots sell other merchant bots here.\n"Beep-boop, dear customer.";;Merchant Bots verkaufen hier andere Merchant Bots.\n�Beep-boop, lieber Kunde.�;;;;;;;;;;;x
+tr_culture_group_murloc_desc;Even if these craftsmans smells fishy, they produce and sell extremely beautiful coralwork.;;Auch wenn diese Handwerker fischig riechen, produzieren und verkaufen sie doch sehr sch�ne Korallen.;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_ciwc_united_localisation.csv]:
```suggestion
tr_culture_group_murloc_desc;Even if these craftsmans smell fishy, they produce and sell extremely beautiful coralwork.;;Auch wenn diese Handwerker fischig riechen, produzieren und verkaufen sie doch sehr sch�ne Korallen.;;;;;;;;;;;x
```

File: localisation/00_ciwc_united_localisation.csv (hunk 35 above)
Comment (MemeWizard) [reply localisation/00_ciwc_united_localisation.csv]: my pun failed

**Comment by MemeWizard** [comment]
@DukeJeans, changed!

**Pull request merged**